app/views/clients/coordinated_entry_assessments/show.haml

```diff
@@ -1,13 +1,11 @@
-- title = "#{_'Coordinated Entry Assessment'} for #{@client.pii_provider(user: current_user).full_name}"
+- title = _'Coordinated Entry Assessment'
 - content_for :title, title
 .o-page
   = render 'clients/breadcrumbs'
   = render 'clients/aliases'
 
 .o-page__header
-  .o-page__title
-    %h1= title
-
+  = render_page_title "#{_'Coordinated Entry Assessment'} for #{@client.pii_provider(user: current_user).full_name}"
```

Comment:
Should these be:
```suggestion
  = render_page_title "#{title} for #{@client.pii_provider(user: current_user).full_name}"
```

just so we don't accidentally get a mis-match in the future?

Reply:
I guess I would lean towards keeping them independent although it is less DRY. I feel like drying it up would be done in another pass and leaving it ugly might make future refactor easier

Reply:
I guess I would lean towards keeping them independent although it is less DRY. I feel like drying it up would be done in another pass and leaving it ugly might make future refactor easier